cernopendata/config.py

```diff
@@ -385,6 +391,8 @@
     "FILES_REST_FILE_URI_MAX_LEN",
     512
 )
+#: Make download availalbe for Root files
+FILES_ROOT_DOWNLOAD_DISABLED = False
```

Comment:
`CERNOPENDATA_DISABLE_DOWNLOADS`?

Please wrap in `os.environ.get` see my earlier comment regarding naming convention https://github.com/cernopendata/opendata.cern.ch/pull/2104#discussion_r156620651